style.css

```diff
@@ -0,0 +1,849 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.right-nav {
+  display: none;
+}
+
+body {
+  max-width: 768px;
+}
```

Comment:
This is the first cause of your `element-not-full-width-in-large-screen` problem. `max-width` doesn't let the element have more width than the one specified in its value. So now `max-width` won't let the `body` have more width than `768px`, the one you specified.